src/mono/wasm/runtime/workers/dotnet-crypto-worker.js

```diff
@@ -379,16 +380,20 @@ function _stringify_err(err) {
     return (err instanceof Error && err.stack !== undefined) ? err.stack : err;
 }
 
-var s_channel;
-
-setup_proxy_console("crypto-worker", console, self.location.origin);
+let s_channel;
 
 // Initialize WebWorker
 onmessage = function (p) {
-    var data = p;
+    /**
+     * @type {any}
```

Comment:
Does this describe `data` or `p`?

Also, is there any value to this comment? Does saying something is `any` have any benefit?

Reply:
It would be much better to rewrite the whole thing in proper typeScript, including the type of the message.
I annotated this just to get rid of type warnings in my editor, but in truth, I didn't try to understand all of the code. 
@eerhardt I believe that you guys contributed most of it ?

Reply:
I made a guess on the types and converted it to typeScript. I also fixed the `data` vs `event` confusion.